R/utils-approved.R

```diff
@@ -0,0 +1,53 @@
+#' Build approved packages and functions tibble
+#'
+#' A utility function to help you build your approve package and functions list.
+#' This can be used by timber to log unnaproved use.
+#'
+#' #' For more details see the help vignette:
+#' \code{vignette("approved", package = "timber")}
+#'
+#' @param pkg_list a named list of character vectors where the name is the
+#' package name with a chacter vector of approved functions or 'All'
+#' @param file (optional) name of file where the approved tibble will be read to.
```

Comment:
Should this have a default Value to show it is optional?  

Default: NULL
Permitted Files: .txt, .csv, ???

![image](https://user-images.githubusercontent.com/10111024/168393877-c78294eb-21dd-4832-aafa-9698fee70133.png)

Reply:
Thanks for the screenshot!  I've updated this to:

```r
#' @param file Name of file where the approved tibble will be read to.
#' If not specified, the tibble is returned.
#'
#' Default: NULL
#'
#' Permitted Files: .RDS
```